plugins/chat/assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -175,6 +205,13 @@ export default Component.extend({
     }
   },
 
+  // TODO (martin) Not ideal....we need this to be trackedarray here
+  // too or ideally just move messages onto the channel
+  @observes("messages")
+  onMessagesChange() {
+    this.livePanel.messages = this.messages;
```

Comment:
Discussed this  problem with you earlier, once we move the `messages` as a tracked array on chatChannel this will not be necessary, the livePanel could store a reference to them easily then